formatter/sarif.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/terraform-linters/tflint/tflint"
 )
 
-func (f *Formatter) sarifPrint(issues tflint.Issues, tferr *tflint.Error, sources map[string][]byte) {
```

Comment:
the unparam linter found that tferr and sources are not being used in the function